.github/workflows/validation_gate.yml

```diff
@@ -0,0 +1,25 @@
+# This looks like a confusing workflow, but it has a purpose (✿ ◕‿◕)
+# It functions kind of as a workflow shim. We can't setup other important workflows to run on pull_request (due to forks), so we setup this workflow to be triggered by pull_request and it is always able to run.
+# When it is finished running, it triggers the other important workflows -- validate_packs, validate_install_plans.
+# Having it setup this way allows us to have manual approval for this workflow run, and then the subsequent triggered runs can access secrets in the repository for validation (even for forked PRs).
+name: Validation Gate
+
+on: 
+  pull_request:
+    branches:
+      - 'main'
```

Comment:
Are we sure we want this to only run on pull requests into `main`? I could see us wanting validation when merging into branches, such as feature branches

Reply:
sound good to me. ill make this change.